deployment/uwsgi/uwsgi.ini

```diff
@@ -5,21 +5,22 @@ buffer-size = 32768
 chmod-socket = 666
 master = true
 processes = 2
-harakiki-verbose = 20
+harakiri-verbose = true
 harakiri = 45
 http-timeout = 45
 socket-timeout = 45
 worker-reload-mercy = 45
 reload-mercy = 45
 mule-reload-mercy = 45
-disable-logging = true
 wsgi-file=/var/www/peregrine/wsgi.py
 plugins = python
 vacuum = true
 uid = www-data
 gid = www-data
 pythonpath = /var/www/peregrine/
 pythonpath = /peregrine/
+logto = /var/log/uwsgi/%n.log
```

Comment:
We should log to `/dev/stderr` to avoid consuming all disk space. (idk why the logging was disabled in the first place)